lib/stripe/subscriptions/invoice.ex

```diff
@@ -264,4 +264,15 @@ defmodule Stripe.Invoice do
     |> put_method(:post)
     |> make_request()
   end
+
+  @doc """
+  Send an invoice
```

Comment:
mind adding this link to the documentation?

https://stripe.com/docs/api/invoices/send